lib/ts/recipe/session/utils.ts

```diff
@@ -41,6 +41,8 @@ export function normaliseSessionConfig(config?: InputType): NormalisedSessionCon
 
     return {
         ...normaliseRecipeModuleConfig(config),
+        // TODO: ideally we'd get the default (or normalized) value from supertokens-website
```

Comment:
Should we care about this TODO? Also, i just happened to bump into this TODO. Is there any other TODO in the comment in this PR?

Reply:
This is the TODO we discussed on call. We should care in the long-term, but it's a fairly low priority fix.
Other than this, no new TODOs have been introduced.